packages/web3/src/signer/signer.ts

```diff
@@ -297,31 +298,6 @@ export abstract class SignerProviderWithMultipleAccounts extends SignerProvider
   abstract setSelectedAccount(address: string): Promise<void>
 }
 
-export class SignerProviderWrapper extends SignerProvider {
```

Comment:
This is removed as the signer should always have access (might be indirectly) to a node provider to build transactions.